apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/User/DomainExistsQuery.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
+
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.User;
+
+public static partial class DomainExistsQuery
+{
+    [Query]
+    public static async Task<bool> DomainExistsAsync(
+        string email,
+        [Service] IMarketParticipantClient_V1 client) =>
```

Comment:
tip: `[Service]` can be omitted from (almost) all service injections. they do no harm, so you don't have to change it. just in case you didn't know 🙂